Header header
geometry_msgs/Pose goal
string command_id
string cart_id
string action
time cancellation_stamp
